### PR TITLE
[ADD][account_cutoff_prepaid] Add move line period on cut off line views

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

**Comment by sbidoul** [comment]
This is useful to have the period of the original move readily available, so users can easily spot "new" moves to be cut-off and review them.

:+1:

File: account_cutoff_prepaid/account_cutoff_view.xml [hunk 1]
```diff
@@ -54,6 +55,7 @@
     <field name="arch" type="xml">
         <field name="parent_id" position="after">
             <field name="move_line_id" invisible="'prepaid' not in context.get('type', '-')"/>
+            <field name="move_line_period_id" invisible="'prepaid' not in context.get('type', '-')"/>
```
Comment (Olivier-LAURENT) [inline account_cutoff_prepaid/account_cutoff_view.xml:58]: Just a little remark here...
With this new field the treeview now contains 4 unstorable fields ! 
No fears about performance ?
No needs to sort the tree on these fields ?

Comment (sbidoul) [inline account_cutoff_prepaid/account_cutoff_view.xml:58]: @Olivier-LAURENT so far, no performance issue even on very large databases, and no user requirement for sorting (yet).

**Comment by Olivier-LAURENT** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**